### PR TITLE
Lucas Alderfer Morse Translator

### PR DESCRIPTION
**Pull request by LucasAlderfer**
I should have done this on a separate branch I think so that I would not be requesting you to pull this into the master branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Shell repo for B1 exercise
 On GitHub
```
Comment (BrianZanti) [inline README.md]: You shouldn't be updating the readme. This indicates to me that you are using `add .` or `add -A` when you are adding files

File: lib/translator.rb [hunk 2]
```diff
@@ -38,5 +41,29 @@ def initialize
...
+                        "7" => "--...",
+                        "8" => "---..",
+                        "9" => "----."}
+
+    create_morse_dictionary
   end
-end
\ No newline at end of file
+
+  def create_morse_dictionary
```
Comment (BrianZanti) [inline lib/translator.rb]: Look at the `.invert` method for Hash

File: lib/translator.rb [hunk 3]
```diff
@@ -38,5 +41,29 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+
+    @num_dictionary = {"0" => "-----",
```
Comment (BrianZanti) [inline lib/translator.rb]: Not sure why you create a separate hash for numbers. The original hash contains numbers

**Comment by ameseee** [comment]
@LucasAlderfer Some great work! Please review the comments and let us know if you have any questions.

**Pull request closed without merging**